src/java.base/share/classes/java/util/random/RandomGenerator.java

```diff
@@ -657,9 +666,13 @@ default int nextInt(int bound) {
      * @throws IllegalArgumentException if {@code origin} is greater than
      *         or equal to {@code bound}
      *
-     * @implSpec The default implementation simply calls
-     * {@link RandomSupport#checkBound(long) checkBound}(bound) and then
-     * {@link RandomSupport#boundedNextInt(RandomGenerator, int) boundedNextInt}(this, bound).
+     * @implSpec The default implementation checks that {@code origin} and
+     * {@code bound} are positive ints. Then invokes {@code nextInt()}, limiting
+     * the result to be greater equal {@code origin} and less than {@code bound}.
+     * If {@code bound} is a power of two then limiting is a simple masking
+     * operation. Otherwise, the result is re-calculated  by invoking
+     * {@code nextInt()} until the result is greater equal {@code origin}
```

Comment:
greater equal -> greater than or equal
and in the overloads below (and above)

Reply:
Updated